### PR TITLE
Fix build for latest core 4.0.0 snapshot

### PR DESCRIPTION
**Pull request by timyates**

### CODE REVIEW

File: itunespodcast/build.gradle [hunk 1]
```diff
@@ -9,6 +9,7 @@ dependencies {
     testAnnotationProcessor mn.micronaut.inject.java
 
     testImplementation(libs.groovy.xml)
+    testImplementation(mn.micronaut.jackson.databind)
```
Comment (sdelamo) [inline itunespodcast/build.gradle:12]: can we use serde instead?

**Review by sdelamo — CHANGES_REQUESTED** [review]

File: itunespodcast/build.gradle (hunk 1 above)
Comment (timyates) [reply itunespodcast/build.gradle:12]: Done

Comment (sdelamo) [reply itunespodcast/build.gradle:12]: should this be ` testImplementation(mn.micronaut.serde.jackson)`?

File: jsonfeed-core/build.gradle [hunk 2]
```diff
@@ -6,6 +6,7 @@ dependencies {
     api mn.jackson.annotations
     api mn.micronaut.inject
     api projects.rssLanguage
+    testImplementation(mn.micronaut.jackson.databind)
```
Comment (sdelamo) [inline jsonfeed-core/build.gradle:9]: should this be ` testImplementation(mn.micronaut.serde.jackson)`?

File: jsonfeed/build.gradle.kts [hunk 3]
```diff
@@ -8,6 +8,7 @@ dependencies {
     api(projects.jsonfeedCore)
     implementation(mn.reactor)
     testAnnotationProcessor(mn.micronaut.inject.java)
+    testImplementation(mn.micronaut.jackson.databind)
```
Comment (sdelamo) [inline jsonfeed/build.gradle.kts:11]: should this be ` testImplementation(mn.micronaut.serde.jackson)`?

**Review by sdelamo — CHANGES_REQUESTED** [review]

**Comment by timyates** [comment]
Dropped Jackson and switched to use Serde, also use the latest setting plugin

**Review by sdelamo — APPROVED** [review]

**Pull request merged**